### ISSUE TITLE
FatalThrowableError in TokenGuard.php line 41:

### ISSUE THREAD

**Issue by emahuni** [rinvex/fort#37]
``` js

FatalThrowableError in TokenGuard.php line 41:
Type error: Argument 1 passed to Illuminate\Auth\TokenGuard::__construct() must be an instance of Illuminate\Contracts\Auth\UserProvider, instance of Rinvex\Fort\Repositories\UserRepository given, called in C:\dev\buck\vendor\laravel\framework\src\Illuminate\Auth\AuthManager.php on line 159

```

This is happening when trying to access api group route:

eg: `'api/user'`

that is:

``` php


    Route::get('/user', function(Request $request)
    {
        return $request->user();
    })->middleware('auth:api');



```

anything in api that uses that middleware throws that error.

**Comment by Omranic** [comment]
API functionality not yet implemented, will make sure to either implement it before first release or handle the exception 👌

**Comment by Omranic** [comment]
Confirmed and fixed in 17e8626d6e11b8e91f250fbbcf4085ea75d23575
